example/lib/main.dart

```diff
@@ -19,24 +20,18 @@ class _MyAppState extends State<MyApp> {
   @override
   void initState() {
     super.initState();
-    initPlatformState();
   }
 
-  // Platform messages are asynchronous, so we initialize in an async method.
   Future<void> initPlatformState() async {
```

Comment:
if you don't use this implementation, please remove it, but not just on this file, from the whole plugin proyect

Reply:
this function is now implemented

Reply:
great, the you should show this information on the screen, like adding a Text that shows the information